pages.pl/android/dalvikvm.md

```diff
@@ -0,0 +1,8 @@
+# dalvikvm
+
+> Wirtualna maszyna Android Java.
+> Więcej informacji: <https://source.android.com/devices/tech/dalvik>.
+
+- Uruchom program Java:
+
+`dalvikvm -classpath {{path/to/file.jar}} {{classname}}`
```

Comment:
```suggestion
`dalvikvm -classpath {{ścieżka/do/pliku.jar}} {{classname}}`
```
[Reference](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)

Reply:
Of course! On it :)

Reply:
The current translation looks fine. Also, we don't translate intent classes and other arguments. We translate simple command arguments like {{class_name}},{{file_name}},etc to their native equivalents also not all arguments require translation in some cases we prefer the English word.